### DISCUSSION TITLE
How to serve static assets at a virtual path?

### DISCUSSION THREAD

**Discussion by realprabs** [adonisjs/core#2750] [Help]
My use case: I want to serve `swagger-ui-dist` npm package assets at `https://mysite.com/docs`. I want to keep serving rest of the files in `public` folder as well.

In general, I am looking for something like below from express:
```js
import { getAbsoluteFSPath } from 'swagger-ui-dist'
app.use('/docs', express.static(getAbsoluteFSPath()))
```

**Comment by RomainLanz** [comment]
Hey @jaysson! 👋 

Any reason to use AdonisJS to make this routing?
This should be handled by your web server, like Nginx, Caddy or Apache.

**Comment by thetutlage** [comment]
Also, I think if you are using the assets of an installed package, then you must use an assets bundler like webpack
